test/fixtures/simple-integration-app/common/models/storetrue.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "storeWithReplaceOnPUTtrue",
```

Comment:
The filename `storetrue` is difficult to understand, could you please use a better name? Also what's the difference between the `stores-replacing.json` and `storetrue.json`?